### PR TITLE
emcute example: Added more FAQ entries to readme

### PR DESCRIPTION
**Pull request by Chamaeleon-**
### Contribution description

I worked with the emcute example and had the added problems (messages send by RSMB don't arrive and topics change ids). I found the solutions in the notes section of the asymcute readme and I think they are needed here too, to be easily accessible for beginners. Especially the the problem with the link local addresses was hard to debug for a RIOT-OS/MQTT-SN beginner.

### Testing procedure
none / Manual proofreading

### Issues/PRs references
none

### CODE REVIEW

File: examples/emcute_mqttsn/README.md [hunk 1]
```diff
@@ -105,3 +105,18 @@ default, this example sets this statically ID to `gertrud`. If you want to
...
 node during compile time. Simply use the `EMCUTE_ID` environment variable for
 this, e.g. build with `EMCUTE_ID=horst make all`.
+
+### I see incomming messages in RSMB, but no outgoing message seems to arrive?
+The UDP socket handling for IPv6 based endpoints in the `Mosquitto.rsmb`
+implementation is buggy when it comes to handling link local addresses,
+as the implementation does not remember the interface on which data comes in,
+hindering it from sending out any responses.
+
+Quick workaround: simply use global addresses
```
Comment (smlng) [inline examples/emcute_mqttsn/README.md]: Minor nit: I would formulate a bit more concise, i.e.

Workaround: either use global IPv6 addresses or [ULAs](https://tools.ietf.org/html/rfc4193).

Comment (Chamaeleon-) [reply examples/emcute_mqttsn/README.md]: Seems like a good idea. I think most important is the explanation why messages get lost. My own workaround was to alter the routing on the linux machine running RSMB.
Is there documentation for a beginner friendly way to add a new address? Maybe it should be linked here too. I found only this makefile flag which is mentioned in this issue https://github.com/RIOT-OS/RIOT/issues/11468 .

**Comment by aabadie** [comment]
@Chamaeleon-, this one seems good to go. Can squash in a single commit please ? And also change the commit message summary to `examples/emcute_mqttsn: add more FAQ entries`.

File: examples/emcute_mqttsn/README.md [hunk 2]
```diff
@@ -105,3 +105,18 @@ default, this example sets this statically ID to `gertrud`. If you want to
 connect more than one node to the broker, you need to set a custom ID for each
 node during compile time. Simply use the `EMCUTE_ID` environment variable for
 this, e.g. build with `EMCUTE_ID=horst make all`.
+
+### I see incomming messages in RSMB, but no outgoing message seems to arrive?
```
Comment (aabadie) [inline examples/emcute_mqttsn/README.md:109]:
```suggestion
### I see incoming messages in RSMB, but no outgoing message seems to arrive?
```

**Comment by miri64** [comment]
@Chamaeleon- please squash to one commit:

```
git rebase -i HEAD~2
```

and mark the last commit as `fixup`

**Comment by aabadie** [comment]
@miri64 I fear that @Chamaeleon- is only using the github web interface (see the branch name). I think we could just open another PR with his commits squashed and merge it right away. What do you think ?

**Comment by miri64** [comment]
> @miri64 I fear that @Chamaeleon- is only using the github web interface (see the branch name). I think we could just open another PR with his commits squashed and merge it right away. What do you think ?

Shouldn't we be able to push to his branch?

**Comment by miri64** [comment]
Alternatively we can just squash and merge once you remove your change request ;-).

**Comment by aabadie** [comment]
> Shouldn't we be able to push to his branch?

I tried and it's forbidden on his fork...

>  we can just squash and merge once you remove your change request

Sure but no merge commit will be created in this case. I don't really care about this merge commit :)

**Comment by miri64** [comment]
Then please ACK or dismiss :-).

**Comment by Chamaeleon-** [comment]
Yes, I tried the web ui... you two are faster than me checking out, sqashing and rebaseing.
Thanks for merging :)

**Comment by miri64** [comment]
@Chamaeleon- thanks for the contribution :-)

**Pull request merged**